src/main/java/org/mockito/internal/stubbing/defaultanswers/ReturnsMoreEmptyValues.java

```diff
@@ -61,13 +36,10 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
         Class<?> returnType = invocation.getMethod().getReturnType();
         return returnValueFor(returnType);
     }
-    
+
     Object returnValueFor(Class<?> type) {
         if (type == String.class) {
             return "";
```

Comment:
@szczepiq Let's create another PR for default Strings, it'll be easier to track for end users as it will appear in the changelog.